swagger.yaml

```diff
@@ -1230,24 +1233,52 @@ definitions:
             title:
               description: "The title of a related dataset"
               type: string
+      related_content:
+        description: "A list of objects containing information of content related to a dataset"
+        type: array
+        items:
+          type: object
+          properties:
+            href:
+              description: "The url to related content"
+              type: string
+            description:
+              description: "The description of the related content"
+              type: string
+            title:
+              description: "The title of the related content"
+              type: string
       release_frequency:
         description: "The release frequency of a dataset"
         type: string
       state:
         $ref: '#/definitions/State'
+      subtopics:
+        description: "A list of ids of sub-topics for a dataset"
+        type: array
+        items:
+          type: "string"
+      survey:
+        description: "The survey for a dataset"
```

Comment:
Is the name of the survey for which the dataset was created from? It might be worth being slightly more descriptive

Reply:
This has now been amended.